box_of_autonomy.md

```diff
@@ -24,6 +24,7 @@ Each team/product will produce/publish/follow the following:
 
 ## Development Time Box
 - Maximum delivery to production time box of 1 week
+- Sprint lengths of 1 week
```

Comment:
- Maximum delivery to production time box of 1 week
was originally a way to enforce regular delivery without constraining to a particular iteration length or style.. we probably should remove it in favor of what was approved/added

Reply:
okay, awesome, I removed it.